### PR TITLE
Update 2.x for Bb 1.3.2

### PR DESCRIPTION
**Pull request by paulfalgout**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ matrix:
     - env: UNDERSCORE=1.5 BACKBONE=1.1
     - env: UNDERSCORE=1.7 BACKBONE=1.2.3
     - node_js: "0.10"
-      env: UNDERSCORE=1.8 BACKBONE=1.2.3
```
Comment (samccone) [inline .travis.yml]: instead of deleting lets just add a new line :)

File: bower.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "upgradeGuide.md"
   ],
   "dependencies": {
-    "backbone": "1.0.0 - 1.2.3",
+    "backbone": "1.0.0 - 1.3.2",
     "underscore": "1.4.4 - 1.8.3",
     "backbone.babysitter": "^0.1.0",
     "backbone.wreqr": "^1.0.0",
```
Comment (maniator) [inline bower.json]: Do we still need the jquery dependency? (on the next line)

Comment (paulfalgout) [inline bower.json]: seems like it should be removed:

https://github.com/marionettejs/backbone.marionette/commit/4c2cc6307a636e9a44ddb8b5385241d8c5429ea1#commitcomment-5873557

File: .travis.yml [hunk 3]
```diff
@@ -14,7 +14,7 @@ matrix:
...
+      env: UNDERSCORE=1.8 BACKBONE=1.3.2
     - env: LODASH=2.4 BACKBONE=1.1
     - env: LODASH=3.0 BACKBONE=1.1
     - env: LODASH=3.1 BACKBONE=1.0
```
Comment (megawac) [inline .travis.yml]: Can we add a lodash env for Backbone@1.3

Comment (samccone) [inline .travis.yml]: 1.3.0 and 1.3.1 were bad releases

Comment (megawac) [inline .travis.yml]: I'm aware, I just want to see `LODASH=3 BACKBONE=1.3.2` or similar

Comment (samccone) [inline .travis.yml]: kk :+1:

**Pull request merged**